src/views/SoraCard.vue

```diff
@@ -1,122 +1,119 @@
-    <sora-card-kyc v-else-if="step === Step.KYC" @go-to-intro="openIntro" />
+    <confirmation-info v-loading="loading" v-if="step === Step.ConfirmationInfo" />
+    <sora-card-intro v-else-if="step === Step.StartPage" @confirm-apply="confirmApply" />
+    <sora-card-kyc v-else-if="step === Step.KYC" @go-to-start="openStartPage" :userApplied="userApplied" />
   </div>
 </template>
 
 <script lang="ts">
 import { mixins } from '@soramitsu/soraneo-wallet-web';
 import SubscriptionsMixin from '@/components/mixins/SubscriptionsMixin';
-import { action } from '@/store/decorators';
+import { action, getter } from '@/store/decorators';
 import { lazyComponent } from '@/router';
 import { Components } from '@/consts';
 
 import { Component, Mixins } from 'vue-property-decorator';
+import { VerificationStatus } from '@/types/card';
 
 enum Step {
-  Intro = 'Intro',
+  StartPage = 'StartPage',
   KYC = 'KYC',
+  ConfirmationInfo = 'ConfirmationInfo',
 }
 
 @Component({
   components: {
     SoraCardIntro: lazyComponent(Components.SoraCardIntroPage),
     SoraCardKyc: lazyComponent(Components.SoraCardKYC),
+    ConfirmationInfo: lazyComponent(Components.ConfirmationInfo),
   },
 })
 export default class SoraCardIntroPage extends Mixins(mixins.LoadingMixin, SubscriptionsMixin) {
+  @getter.soraCard.currentStatus private currentStatus!: VerificationStatus;
+
   @action.pool.subscribeOnAccountLiquidityList private subscribeOnAccountLiquidityList!: AsyncFnWithoutArgs;
   @action.pool.subscribeOnAccountLiquidityUpdates private subscribeOnAccountLiquidityUpdates!: AsyncFnWithoutArgs;
+  @action.soraCard.getUserStatus private getUserStatus!: AsyncFnWithoutArgs;
   @action.soraCard.subscribeToTotalXorBalance private subscribeToTotalXorBalance!: AsyncFnWithoutArgs;
   @action.soraCard.unsubscribeFromTotalXorBalance private unsubscribeFromTotalXorBalance!: AsyncFnWithoutArgs;
   @action.pool.unsubscribeAccountLiquidityListAndUpdates private unsubscribeLPUpdates!: AsyncFnWithoutArgs;
 
-  step: Step = Step.Intro;
-  isUserPassedKyc = false;
+  step: Nullable<Step> = null;
+  userApplied = false;
 
   Step = Step;
 
-  confirmApply(): void {
+  confirmApply(userApplied: boolean): void {
+    this.userApplied = userApplied;
     this.step = Step.KYC;
   }
 
-  openIntro(): void {
-    this.step = Step.Intro;
+  openStartPage(withoutCheck: boolean): void {
+    if (withoutCheck) {
+      this.step = Step.ConfirmationInfo;
+      return;
+    }
+
+    this.checkKyc();
   }
 
-  async created(): Promise<void> {
-    if (this.isUserPassedKyc) {
-      this.step = Step.KYC;
+  async checkKyc(): Promise<void> {
+    await this.getUserStatus();
+
+    if (this.currentStatus) {
+      this.step = Step.ConfirmationInfo;
+    } else {
+      this.step = Step.StartPage;
     }
+  }
 
+  async created(): Promise<void> {
     this.setStartSubscriptions([
+      this.subscribeToTotalXorBalance,
       this.subscribeOnAccountLiquidityList,
```

Comment:
do we need `this.subscribeOnAccountLiquidityList` and `this.subscribeOnAccountLiquidityUpdates` here?

Reply:
probably not